src/components/Tree/Tree.tsx

```diff
@@ -129,22 +47,14 @@ export const Tree: FC<TreeProps> = ({
     }, []);
 
     return (
-        <ul
-            id={treeId}
-            data-test-id="tree"
-            className="tw-p-0 tw-m-0 tw-font-sans tw-font-normal tw-list-none tw-text-left tw-text-sm tw-select-none"
-        >
-            <DndWrapper id={treeId}>
-                {renderNodeArray({
-                    nodes: treeNodes,
-                    treeId,
-                    activeIds,
-                    onClick: onNodeClick,
-                    onDrop: handleDrop,
-                    onEditableSave: handleEditableSave,
-                    onNodeExpand: handleNodeExpand,
-                })}
-            </DndWrapper>
-        </ul>
+        <TreeContext.Provider value={{ treeId: id, selectedIds, onSelect: handleSelect, draggable }}>
+            <ul
+                id={id}
+                data-test-id="tree"
+                className="tw-p-0 tw-m-0 tw-font-sans tw-font-normal tw-list-none tw-text-left tw-text-sm tw-select-none"
+            >
+                <DndWrapper id={id}>{children}</DndWrapper>
```

Comment:
Why pass the same `id` to `ul` element and `DndWrapper`?

Reply:
I don't really see a reason for them to be different